src/app/tests/suites/certification/Test_TC_DRLK_2_9.yaml

```diff
@@ -90,9 +90,8 @@ tests:
           saveAs: NumberOfTotalUsersSupported
           constraints:
               minValue: 0
-              maxValue: 255
+              maxValue: 65534
```

Comment:
Why is this 65534?  I don't see that constraint in the spec.

Reply:
@bzbarsky-apple it is mentioned in testplan: valid number in the inclusive range of 0 to 65534
![image](https://user-images.githubusercontent.com/102359958/183102512-ca5b8aea-1e0a-47d0-bacf-59226e994af3.png)